tests/apollo/util/skvbc_history_tracker.py

```diff
@@ -1074,6 +1099,32 @@ async def run_concurrent_ops(self, num_ops, write_weight=.70):
             sent += len(clients)
         return read_count, write_count
 
+    async def run_concurrent_conflict_ops(self, num_ops, write_weight=.70):
+        if self.no_conflicts is True:
+            print("call to run_concurrent_conflict_ops with no_conflicts=True, calling run_concurrent_ops instead")
+            await self.run_concurrent_ops(num_ops, write_weight)
+            return
+        max_concurrency = len(self.bft_network.clients) // 2
+        max_size = len(self.skvbc.keys) // 2
+        sent = 0
+        write_count = 0
+        read_count = 0
+        while sent < num_ops:
+            readset = self.readset(0, max_size)
+            writeset = self.writeset(max_size, readset)
+            read_version = 0
+            clients = self.bft_network.random_clients(max_concurrency)
+            async with trio.open_nursery() as nursery:
+                for client in clients:
+                    if random.random() < write_weight:
+                        nursery.start_soon(self.send_tracked_kv_set, client, readset, writeset, read_version)
+                        write_count += 1
+                    else:
+                        nursery.start_soon(self.send_tracked_read, client, max_size)
+                        read_count += 1
+            sent += len(clients)
+        return read_count, write_count
```

Comment:
why not do the same refactoring here too?

Reply:
I mean, to extract `send_concurrent_ops()`...

Reply:
done